sync-files/sync.yml

```diff
@@ -162,6 +163,23 @@ group:
   repos: |
     ecmwf/anemoi-utils
 
+# Anemoi specific
+- files:
+  - source: sync-files/anemoi/all/
+    dest: ./
+    template:
+      repo_name: anemoi
+      year: 2024-2025
+      authors: Anemoi Contributors
+      short_desc: "Miscellanous tools for training data-driven weather forecasts."
+    exclude: |
+      .release-please-manifest.json
+      .github/workflows/push-to-private.yml
+      .github/workflows/downstream-ci-hpc.yml
+      .github/workflows/python-publish.yml
+      .github/workflows/release-please.yml
+  repos: |
+    ecmwf/anemoi
 
 # Anemoi Core specific
 - files:
```

Comment:
The `pr-label-ats` needs to be added here as well, so that in the future it will sync to core if it's ever changed.